staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation_test.go

```diff
@@ -7602,7 +7603,8 @@ func TestValidateCustomResourceDefinitionValidation(t *testing.T) {
 							XListType: strPtr("atomic"),
 							Items: &apiextensions.JSONSchemaPropsOrArray{
 								Schema: &apiextensions.JSONSchemaProps{
-									Type: "string",
+									Type:      "string",
+									MaxLength: int64ptr(10),
```

Comment:
just out of curiosity, did you have to constrain the length to get the cost within an allowed range?

Reply:
I believe most (all?) of these were added as a result of tests failing due to cost limits being exceeded.  I consider it a feature to have the tests setting max lengths since that's a thing we want to encourage :grin:

Reply:
Sure, sounds good. Hopefully constraining the maxItems would work as well (and would probably be what we did in real life, maybe along with constraining the string field length, but probably not to 10 chars)